---
title: 57396 - ThrowingException
ms.custom: 
ms.date: 03/30/2017
ms.prod: .net-framework
ms.reviewer: 
ms.suite: 
ms.technology: dotnet-clr
ms.tgt_pltfrm: 
ms.topic: article
ms.assetid: 4f865de5-62c1-443c-9dcd-1675ca127334
caps.latest.revision: "3"
author: dotnet-bot
ms.author: dotnetcontent
manager: wpickett
ms.openlocfilehash: 08022bf0aca169fdc1a00cf5d98acd5b69bdab94
ms.sourcegitcommit: ce279f2d7fe2220e6ea0a25a8a7a5370ddf8d9f0
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 12/02/2017
---
# <a name="57396---throwingexception"></a>57396 - ThrowingException
## <a name="properties"></a>Propriet√†  
  
|||  
|-|-|  
|ID|57396|  
|Parole chiave|Infrastruttura|  
|Livello|Avviso|  
|Canale|Microsoft-Windows-Application Server-Applications/Analytic|  
  
## <a name="description"></a>Descrizione  
 Questo evento viene generato in occasione di un'eccezione.  
  
## <a name="message"></a>Messaggio  
 Generazione di un'eccezione. Origine %1.  
  
## <a name="details"></a>Dettagli
